doc-content/enterprise-only/admin-and-config/setting-the-priority-of-task-MVEL-proc.adoc

```diff
@@ -0,0 +1,16 @@
+[id='setting-the-priority-of-task-MVEL-proc']
+= Setting the priority of a task as an MVEL expression
+
+You can set the priority of a task as an MVEL expression in {CENTRAL} from the *Task Inbox* page. Note that all users may not have permissions for setting priority of a task.
+
+.Prerequisites
+You have created a project in {CENTRAL}.
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Track* -> *Task Inbox*.
+. On the *Task Inbox* page, click the task to open it.
```

Comment:
[Style question] - might it be better to refer to the task UI element or perhaps rephrase to "To do this, do that":

s/click the task to open it.
/click the name of the task you want to open.
/click the task name to open the task.
/to open a specific task, click the task name.